ext_mongodb.php

```diff
@@ -227,6 +333,57 @@ public function __wakeup() : void;
 
 	<<__Native>>
 	public function selectServer(ReadPreference $readPreference): Server;
+
+/* {{{ APM */
+	public function addSubscriber( Monitoring\Subscriber $subscriber ) : void
+	{
+		$this->subscribers[] = $subscriber;
```

Comment:
I did think about it, but didn't consider this to be a *problem*. I guess I can change this, but is it important enough?